src/System.IO.FileSystem/tests/File/Copy_str_str.cs

```diff
@@ -1,439 +1,150 @@
-                    {
-                        iCountErrors++;
-                        printerr("Error_98yv7! Expected==" + cWriteArr[i] + ", got value==" + cReadArr[i]);
-                    }
-                }
-                sr2.Dispose();
-                fileStream.Dispose();
-
-                fil1.Delete();
-                fil2.Delete();
-            }
-            catch (Exception exc)
-            {
-                iCountErrors++;
-                printerr("Error_28vc8! Unexpected exception, exc==" + exc.ToString());
             }
+        }
 
-/* Scenario disabled while porting because it modifies state outside the test's working directory
-#if !TEST_WINRT
-            String tmpFile;
-            // [] Interesting case:
-
-            strLoc = "Loc_478yb";
-
-            new FileStream(filName, FileMode.Create).Dispose();
-            File.Delete("\\TestFile.tmp");
-            fil2 = new FileInfo(filName);
-
-            //Cleanup root files
-            tmpFile = "..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\..\\TestFile.tmp";
-            File.Copy(fil2.Name, tmpFile);
-            fil1 = new FileInfo(fil2.FullName.Substring(0, fil2.FullName.IndexOf("\\") + 1) + fil2.Name);
-            iCountTestcases++;
-            if (!fil1.FullName.Equals(fil2.FullName.Substring(0, fil2.FullName.IndexOf("\\") + 1) + fil2.Name))
-            {
-                iCountErrors++;
-                Console.WriteLine("[{0}][{1}]", fil1.FullName, Path.GetFullPath(tmpFile));
-                printerr("Error_298gc! Incorrect fullname set during copy");
-            }
-            new FileInfo(filName).Delete();
-            fil1.Delete();
-            if (File.Exists(tmpFile))
-                File.Delete(tmpFile);
-#endif
-*/
-
-            // [] Vanilla copy operation
-
-            strLoc = "Loc_090jd";
-            destFile = Path.Combine(TestInfo.CurrentDirectory, Path.GetRandomFileName());
-
-            File.Create(filName).Dispose();
-            fil2 = new FileInfo(filName);
-            iCountTestcases++;
-            try
+        [Fact]
+        [PlatformSpecific(PlatformID.AnyUnix)]
+        [ActiveIssue(2205)]
+        public void UnixWhitespacePath()
+        {
+            string testFile = GetTestFilePath();
+            using (File.Create(testFile))
             {
-                File.Copy(filName, destFile, true);
-                fil1 = new FileInfo(destFile);
-                fil2 = new FileInfo(filName);
-                if (!File.Exists(fil1.FullName))
-                {
-                    iCountErrors++;
-                    printerr("Error_378ce! File not copied");
-                }
-                if (!File.Exists(fil2.FullName))
+                foreach (string whitespace in WhitespaceVariations)
                 {
-                    iCountErrors++;
-                    printerr("Error_t948v! Source file gone");
+                    Copy(testFile, whitespace);
```

Comment:
Also, this is copying to a file in the current directory named for whatever is in whitespace... that's going to leave behind temporary files, and likely cause problems when this method runs as part of one of the other derived types and tries to copy to a file that already exists.